BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -175,12 +176,19 @@ extern BOXRepresentationStatus *const BOXRepresentationStatusPending;
 extern BOXRepresentationStatus *const BOXRepresentationStatusNone;
 extern BOXRepresentationStatus *const BOXRepresentationStatusError;
 
-// Representation dimensions
-typedef NSString BOXRepresentationDimensions;
-extern BOXRepresentationDimensions *const BOXRepresentationDimensionsThumbnail;
-extern BOXRepresentationDimensions *const BOXRepresentationDimensionsLargeThumbnail;
-extern BOXRepresentationDimensions *const BOXRepresentationDimensions1024x1024;
-extern BOXRepresentationDimensions *const BOXRepresentationDimensions2048x2048;
+// Representation Supported Image Formats
+// Formats greater than 1024 are rendered on demand, all others are rendered once the file is uploaded to Box.
+// JPG 2048, PNG 1024 and PNG 2048 representations not available for video file types
+// If the original file is not a square, images requested above 160x160 will retain the file's original aspect ratio.
+// JPG avaiable in all sizes, PNG available only in 1024 and 2048
+
+typedef NSString BOXRepresentationImageDimensions;
+extern BOXRepresentationImageDimensions *const BOXRepresentationImageDimensionsJPG32;
```

Comment:
Any reason to add `JPG` in these names?

Reply:
These dimensions are only available for jpeg only. The other 2 largest dimensions are both jpeg or png. See above comments.